HW7/Mengyuan_HW7.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import numpy as np
+import pandas as pd 
+import matplotlib.pyplot as plt
```

Comment:
Both numpy and matplotlib are unused imports.  They can be deleted from the script.